src/hooks/api/account/useActiveAccount.ts

```diff
@@ -17,9 +17,13 @@ const useActiveAccount = () => {
             ? {
                   ...activeAccount,
                   balance: balanceData?.accounts?.[activeAccount?.loginid]?.balance ?? 0,
+                  country: authData?.authorize?.country,
+                  isWalletAccount:
+                      activeAccount.account_category === 'wallet' ||
+                      authData?.authorize?.linked_to?.some(item => item.platform === 'dwallet'),
```

Comment:
We still need to check for the dwallet because the user migrated their account to wallets. However for new users who directly created a new account (no migration), they wont see the dwallet value. Also if the user migrated their account to a wallet, the `account_category` still stays as trading

![Screenshot 2025-03-13 at 5 37 54 PM](https://github.com/user-attachments/assets/f4c898e8-0981-4f3b-bc67-577c1b859586)